.github/pull_request_template.md

```diff
@@ -18,3 +18,7 @@ Assist reviewers with steps they can take to test that the PR does what it says
 ## 📸 Screenshot
 
 Add a screenshot (if possible).
+
+## 🚀 Deployment Notes
```

Comment:
Secondly, and of utmost importance 
Rocket... Or 
🚢 ship
:shipit: shipit
🛫 departure
🚚 truck
🖥️ ⌨️  computer and keyboard

Reply:
truck ftw!

Reply:
🚚, see 152a3b0a94ee80ac091b9841125802617d46f8e7.